server/routes/userRoutes.js

```diff
@@ -11,5 +11,5 @@ const userRoutes = express.Router();
 userRoutes.post('/auth/signup', validateSignUp, userController.signUp);
 userRoutes.post('/auth/login', validateLogin, userController.login);
 userRoutes.put('/verify/:token', auth, userController.verify);
-userRoutes.get('/all', auth, userController.allUsers);
+userRoutes.get('/', auth, userController.allUsers);
```

Comment:
I think it would also be nice if you could destructure the `userController` object to have direct access to its methods in this file. This would help provide a clean and less repetitive code.